test/test_minipyro.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 import torch
 from pyro.generic import distributions as dist
-from pyro.generic import infer, optim, pyro, pyro_backend
+from pyro.generic import handlers, infer, optim, pyro, pyro_backend
```

Comment:
You can probably start importing this from pyroapi now (since it is a test dependency)?

Reply:
fixed.